src/app/portlets/content-types/edit/content-types-edit.component.html

```diff
@@ -13,7 +13,8 @@ <h4>{{ data.name }} </h4>
             </div>
         </div>
         <div class="content-type__edit-actions">
-            <button type="button" pButton secondary label="Edit" (click)="show = true" icon="fa-edit" id="form-edit-button"></button>
+            <button type="button" pButton secondary label="{{ dotMessageService.get('contenttypes.action.edit') }}" (click)="show = true" icon="fa-edit" id="form-edit-button"></button>
+            <button type="button" pButton label="{{ dotMessageService.get('contenttypes.dropzone.rows.add') }}" icon="fa-plus" (click)="dotEventsService.notify('add-row')" id="form-add-row"></button>
```

Comment:
maybe create new buttons that use md icons?

Reply:
Can't we just modify the `dot-icon-button` to receive a param of label?

Reply:
We can use the `dot-icon-button`, maybe rename it to `dot-button` and it will receive 2 params: icon and label. If receive just icon we do as it right now if receive both we do icon + text